src/js/angular/clustermanagement/templates/modal/add-nodes-dialog.html

```diff
@@ -0,0 +1,67 @@
+<!-- Start add nodes to cluster modal -->
```

Comment:
what are these comments?!

Reply:
I saw them somewhere else. They are for consistency :)

Reply:
removed them